internal/pkg/levee/levee.go

```diff
@@ -69,15 +77,32 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
-func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
+func reportSourcesReachingSink(conf *config.Config, pass *analysis.Pass, suppressedNodes suppression.ResultType, propagations map[*source.Source]propagation.Propagation, sink ssa.Instruction) {
 	for src, prop := range propagations {
-		if prop.IsTainted(sink) {
+		if prop.IsTainted(sink) && !isSuppressed(sink.Pos(), suppressedNodes, pass) {
 			report(conf, pass, src, sink.(ssa.Node))
 			break
 		}
 	}
 }
 
+func isSuppressed(pos token.Pos, suppressedNodes suppression.ResultType, pass *analysis.Pass) bool {
+	for _, f := range pass.Files {
+		if pos < f.Pos() || f.End() < pos {
+			continue
+		}
+		path, _ := astutil.PathEnclosingInterval(f, pos, pos)
+		if len(path) < 2 {
+			return false
+		}
```

Comment:
This logic was not obvious to me before pulling up the `astutil` documentation.  Consider adding the comment `// Get all ast ancestors of this pos`.  Consider renaming `path` to `ancestors` for consistency with doc.

Reply:
I personally prefer `path` because the function is called `Path...`, so it feels like a natural name.

I've added a comment which I think describes the behavior in sufficient detail for our use case.

WDYT?

Reply:
> I personally prefer path because the function is called Path..., so it feels like a natural name.

Fair enough.  The clarifying comment was the real ask, and we have one now.  👍